common/tools/eslint-plugin-azure-sdk/package.json

```diff
@@ -24,20 +24,27 @@
   "bugs": {
     "url": "https://github.com/Azure/azure-sdk-for-js/issues"
   },
-  "main": "dist/index.js",
+  "main": "dist-esm/src/index.js",
+  "exports": {
```

Comment:
We should be using tshy here to ensure we have proper types for CJS and ESM.  Otherwise these are only types for ESM and not CJS